macros/mySensorInfo.py

```diff
@@ -4,62 +4,63 @@
 ################################################################################
 ##################################  Geometry  ##################################
 # {<sensor name_long>, <sensor name_short/label>, <pitch [um]>, <width [um]>, <length [mm]>,
-#  <Bias voltage [V]>, <thickness [um]>, <resistivity [Ohm/sq]>, <capacitance [pF/mm2]>, tag}
+#  <Bias voltage [V]>, <thickness [um]>, <resistivity [Ohm/sq]>, <capacitance [pF/mm2]>, tag, central-ch-position}
```

Comment:
How are these "central channel position" numbers gotten? Are they coming from a macro or taken from Geometry?

Reply:
Ok! I like that idea! I'll work on that ;)